.wokeignore

```diff
@@ -0,0 +1,4 @@
+lib/charms/nginx_ingress_integrator/v0/nginx_route.py
```

Comment:
Looks like we have both wokieignore` and `ignore` in `woke.yaml`?

Reply:
Related: #49